tools/packaging/kata-deploy/kata-deploy/overlays/stable/env_stable_conf.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata: 
+  name: kata-deploy
+  namespace: kube-system
+spec:
+  template:
+    spec:
+      containers:
+      - name: kube-kata
+        env:
+          - name: SHIMS
+            value: "clh dragonball fc qemu qemu-nvidia-gpu qemu-sev qemu-snp qemu-tdx"
```

Comment:
The value in [kata-deploy-stable.yaml](https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-deploy/base/kata-deploy-stable.yaml#L35) has changed since this PR was created. It is now
`"clh cloud-hypervisor dragonball fc qemu qemu-nvidia-gpu qemu-sev qemu-snp qemu-tdx"`.